src/widgets/header/index.tsx

```diff
@@ -0,0 +1,127 @@
+import AppBar from '@mui/material/AppBar';
+import Box from '@mui/material/Box';
+import Toolbar from '@mui/material/Toolbar';
+import IconButton from '@mui/material/IconButton';
+import Typography from '@mui/material/Typography';
+import Menu from '@mui/material/Menu';
+import MenuIcon from '@mui/icons-material/Menu';
+import Container from '@mui/material/Container';
+import Button from '@mui/material/Button';
+import MenuItem from '@mui/material/MenuItem';
+import { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+
+interface IHeaderProps {}
+
+const Header: React.FunctionComponent<IHeaderProps> = (props) => {
+  const [anchorElNav, setAnchorElNav] = useState<null | HTMLElement>(null);
+  const navigate = useNavigate();
+
+  const handleOpenNavMenu = (event: React.MouseEvent<HTMLElement>) => {
+    setAnchorElNav(event.currentTarget);
+  };
+  const handleCloseNavMenu = () => {
+    setAnchorElNav(null);
+  };
+
+  const navToProfile = () => {
+    handleCloseNavMenu();
+    navigate('/profile');
+  };
+  const navToEdit = () => {
```

Comment:
linter should warn about this - no return type

Reply:
fixed